src/pages/experimental/edit-pattern/index.mdx

```diff
@@ -0,0 +1,197 @@
+---
+title: Edit Pattern
+---
+
+<PageDescription>
+
+Editing is making changes to a resource that already exists in the system.
+Updating and editing should use the same behaviour as when the resource is created. This should only be varied in situations where the number of editable fields is significantly smaller or larger than during creation, or the context of current settings is required.
+
+Warn the user when the consequence of editing could damage resources or connections in their system.
```

Comment:
According to our design team, we can't have line breaks in page descriptions. As we address consequences and related actions on each variant, we can probably delete this line. What do you think?

Reply:
Yes, I think we can cope with that